demisto_sdk/__main__.py

```diff
@@ -682,6 +687,12 @@ def validate(ctx, config, file_paths: str, **kwargs):
     """Validate your content files. If no additional flags are given, will validated only committed files."""
     from demisto_sdk.commands.validate.validate_manager import ValidateManager
 
+    if os.getenv(ENV_SDK_WORKING_OFFLINE, "").lower() == "yes":
+        logger.info(
+            "[red]no connection[/red]"
+        )
+        sys.exit(1)
```

Comment:
do not use sys.exit. Raise an exception instead. No need to log in case of an exception